packages/react-draggable-dialog/src/components/DraggableDialogSurface/DraggableDialogSurface.tsx

```diff
@@ -9,27 +13,25 @@ import { useDraggableDialogSurface } from './useDraggableDialogSurface';
  * DraggableDialogSurface is a wrapper around the DialogSurface component that,
  * when composed with DraggableDialog, can be dragged.
  */
-export const DraggableDialogSurface: React.FC<DraggableDialogSurfaceProps> = ({
-  children,
-  className,
-  ...props
-}) => {
-  const styles = useStyles();
-  const { ref, style, mountNode } = useDraggableDialogSurface(props);
+export const DraggableDialogSurface: ForwardRefComponent<DraggableDialogSurfaceProps> =
+  React.forwardRef((props, _ref) => {
```

Comment:
The name `ref` is already used (the constant declared on line 20), so it can't be used again.

However, I could rename `_ref` to something else if you think that's better -- e.g. something like `forwardedRef`?

Reply:
yup, makes sense, we tend to use variable names prefixed with "_" if they are not used (but the function signature requires you to pass it). `forwardedRef` would be great.

Reply:
Thanks! I have now updated the names.